test/test_hpydict.py

```diff
@@ -33,7 +33,7 @@ def test_New(self):
         """)
         assert mod.f() == {}
 
-    def test_SetItem(self):
+    def test_set_item(self):
```

Comment:
do we actually need these tests? At a very quick glace it seems to me that they are not testing anything more than the equivalent tests in `test_object.py`.
If the don't, we can just kill :)
If they **do** test more, we should improve `test_object.py` and kill these.

Reply:
They're here to test that one can re-implement the equivalent of PyDict_GetItem and PyDict_SetItem easily. :)